OsuPlayer/Views/PlaylistViewModel.cs

```diff
@@ -141,7 +144,10 @@ private void RefreshAllIcons()
     {
         if (_materialIcons.Count != Playlists?.Count || Player.RepeatMode.Value != RepeatMode.Playlist) return;
 
-        for (var i = 0; i < _materialIcons.Count; i++) _materialIcons[i].IsVisible = Player.SelectedPlaylist.Value?.Id == Playlists?[i].Id;
+        Dispatcher.UIThread.Post(() =>
+        {
+            for (var i = 0; i < _materialIcons.Count; i++) _materialIcons[i].IsVisible = Player.SelectedPlaylist.Value?.Id == Playlists?[i].Id;
```

Comment:
A line break here for sure. Line is to long. I like to keep those one liner ifs, fors, foreaches only for really short stuff